### ISSUE TITLE
More Info Screen - VoiceOver Does Not read Reports Fact Sheet PDF Headings

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor#76]
PDF's are hard to read. VoiceOver is not reading any of the headings in the document which is iOS and Android issues with PDFs. Plus, the text is small and difficult to read.

**Comment by carolynhuang** [comment]
Please close out. Fact sheets no longer exist in app.

**Comment by jfc3-dol** [comment]
Looks like it's been removed.

Closing issue.
